.github/workflows/linux.yml

```diff
@@ -24,29 +24,17 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  CCACHE_COMPRESS:      "true"
-  CCACHE_COMPRESSLEVEL: "6"
-  CCACHE_DIR:           "/dev/shm/.ccache"
-  CCACHE_DIRECT:        "true"
-  CCACHE_MAXSIZE:       "128M"
-  CCACHE_SLOPPINESS:    "pch_defines,time_macros"
-  CCACHE_UMASK:         "002"
-
-  CLAMDB_DIR:      "/var/lib/clamav"
-
+  # Docker commmand to run if performing cross builds.defaults:
+  # The command remnoves the container on exit, runs the 
```

Comment:
typo: removes

Reply:
I reworded this comment slightly as well. It was mentioning `HOME` directory, instead of environment.